gui/qt/i18n/fr_FR.ts

```diff
@@ -261,12 +261,12 @@
     <message>
         <location filename="../mainwindow.ui" line="147"/>
         <source>Step</source>
-        <translation></translation>
+        <translation>Pas-à-pas détaillé</translation>
```

Comment:
Moui en fait non, ça rentre pas niveau UI, déjà que c'est limite en anglais...
Je vais voir si on peut pas simplement avoir cette phrase là en tant que tooltip, plutot.

```suggestion
        <translation></translation>
```